### PR TITLE
[SEO] Add important meta tag information

### PR DESCRIPTION
**Pull request by mikedelgaudio**
This PR involves adding the following`<meta>` tag properties to the `index.tsx` to enhance SEO:

## Changes:
- Revised website meta description with a fuller website description
- Added meta keywords to aid in SEO results
- Added `og` meta tags to aid in social media and alternative platform awareness. This includes `title`, `type`, `image`, `image:alt`, `image:type`, `image:width`, `image:height`, and `image:description`. @surma @jakearchibald however please let me know if this may have a poor effect to site performance

### CODE REVIEW

File: src/static-build/pages/index/index.tsx [hunk 1]
```diff
@@ -27,7 +28,25 @@ const Index: FunctionalComponent<Props> = () => (
       <title>Squoosh</title>
       <meta
         name="description"
-        content="Compress and compare images with different codecs, right in your browser"
+        content="Squoosh is the ultimate image optimizer that allows you to compress and compare images with different codecs in your browser."
+      />
+      <meta
+        name="keywords"
+        content="best image optimizer, png to webp, online image compressor, image optimizer, online image optimizer, compress image, jpg to webp, shrink image size"
```
Comment (nucliweb) [inline src/static-build/pages/index/index.tsx]: Don't yo forget the AVIF image format 😜

```suggestion
        content="best image optimizer, png to webp, online image compressor, image optimizer, online image optimizer, compress image, jpg to webp, jpg to avif, shrink image size"
```

**Review by nucliweb — APPROVED** [review]

**Review by nucliweb — APPROVED** [review]

**Review by surma — COMMENTED** [review]
Woah, I can’t believe we never added social

File: src/static-build/pages/index/index.tsx [hunk 2]
```diff
@@ -27,7 +28,25 @@ const Index: FunctionalComponent<Props> = () => (
...
+      />
+      <meta property="og:title" content="Squoosh" />
+      <meta property="og:type" content="website" />
+      <meta property="og:image" content={ogImage} />
+      <meta property="og:image:type" content="image/png" />
+      <meta property="og:image:width" content="500" />
+      <meta property="og:image:height" content="500" />
+      <meta
+        property="og:image:alt"
+        content="A cartoon of a hand squeezing an image file on a dark background."
```
Comment (surma) [inline src/static-build/pages/index/index.tsx:47]: For some reason, the alt isn’t actually recognized by Facebook?

<details>

Screen Shot 2021-07-19 at 12 21 06


</details>

**Review by surma — CHANGES_REQUESTED** [review]
Love the keyword changes. 

While we are on the task on adding social, would you mind also adding [Twitter Cards](https://developer.twitter.com/en/docs/twitter-for-websites/cards/guides/getting-started)?

File: src/static-build/pages/index/index.tsx (hunk 2 above)
Comment (mikedelgaudio) [reply src/static-build/pages/index/index.tsx:47]: Interesting, let me research this a bit more

Comment (mikedelgaudio) [reply src/static-build/pages/index/index.tsx:47]: @surma sadly seems like a relatively undocumented issue. (also, many devs forget to put an alt) Here's one possibility I found:

- The Facebook crawler has a bug. In 2020, users discovered that Facebook's crawler could not decipher between `og:image` and `og:image:secure_url`. I think it is worth trying again with `secure_url` and let me know if that helps. You also may have experienced a FB caching issue which seems fairly common on the forums I found. More info: https://stackoverflow.com/questions/8855361/fb-opengraph-ogimage-not-pulling-images-possibly-https

**Comment by mikedelgaudio** [comment]
> Love the keyword changes.
> 
> While we are on the task on adding social, would you mind also adding [Twitter Cards](https://developer.twitter.com/en/docs/twitter-for-websites/cards/guides/getting-started)?

All set :)

File: src/static-build/pages/index/index.tsx [hunk 3]
```diff
@@ -27,7 +28,28 @@ const Index: FunctionalComponent<Props> = () => (
...
         name="description"
-        content="Compress and compare images with different codecs, right in your browser"
+        content="Squoosh is the ultimate image optimizer that allows you to compress and compare images with different codecs in your browser."
+      />
+      <meta
+        name="keywords"
+        content="best image optimizer, png to webp, online image compressor, image optimizer, online image optimizer, compress image, jpg to webp, jpg to avif, shrink image size"
+      />
+      <meta name="twitter:card" content="summary" />
+      <meta name="twitter:site" content="@SquooshApp" />
```
Comment (surma) [inline src/static-build/pages/index/index.tsx:34]:
```suggestion
      <meta name="twitter:site" content="@SquooshApp" />
      <meta name="twitter:image" content={ogImage} />
```

**Review by surma — CHANGES_REQUESTED** [review]
Almost there! (I added a `twitter:image` tag btw)

One thing that both Facebook and Twitter require is that the URLs for image are _absolute_ URLs. So we need to somehow figure out our host name and ad that. Not sure if we have a mechanism in place. @jakearchibald can you potentially help out here?

**Comment by mikedelgaudio** [comment]
> Almost there! (I added a `twitter:image` tag btw)
> 
> One thing that both Facebook and Twitter require is that the URLs for image are _absolute_ URLs. So we need to somehow figure out our host name and ad that. Not sure if we have a mechanism in place. @jakearchibald can you potentially help out here?

Ah yeah, I also found that Twitter relies on the `og:image` as fallback for `twitter:image` (https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/markup). I'm curious if defining it explicitly with `twitter:image` helps performance or is it writing the same code twice?

**Comment by surma** [comment]
> I also found that Twitter relies on the og:image as fallback for twitter:image

Did not know that. Let’s remove my addition again, then, lol.

File: src/static-build/pages/index/index.tsx [hunk 4]
```diff
@@ -27,7 +28,28 @@ const Index: FunctionalComponent<Props> = () => (
       <title>Squoosh</title>
       <meta
         name="description"
-        content="Compress and compare images with different codecs, right in your browser"
+        content="Squoosh is the ultimate image optimizer that allows you to compress and compare images with different codecs in your browser."
+      />
+      <meta
+        name="keywords"
+        content="best image optimizer, png to webp, online image compressor, image optimizer, online image optimizer, compress image, jpg to webp, jpg to avif, shrink image size"
+      />
```
Comment (atjn) [inline src/static-build/pages/index/index.tsx]:
```suggestion
```
Sorry, but I don't think `keywords` will do anything other than increase the download size for users. I'm happy to discuss why, but I think this article covers everything: https://yoast.com/meta-keywords/

**Review by atjn — CHANGES_REQUESTED** [review]

File: src/static-build/pages/index/index.tsx (hunk 4 above)
Comment (mikedelgaudio) [reply src/static-build/pages/index/index.tsx]: Thanks for resource and new learnings!

**Review by atjn — APPROVED** [review]

**Comment by mikedelgaudio** [comment]
@atjn I believe you may need to sign the CLA if possible

**Comment by atjn** [comment]
I have already signed the CLA. I believe this is a known issue with the google-cla bot, and that @surma can still merge this despite the error :)

**Comment by jakearchibald** [comment]
I'll figure out something for the absolute URLs. I need this for my blog too 😄

**Comment by jakearchibald** [comment]
@surma I think that does it. Could you review my change?

**Review by surma — APPROVED** [review]
- [x] Absolute image URLs in markup
- [x] Twitter Cards validator is showing the image
- [x] Facebook Graph Debug Tool is showing the image
- [x] Facebook Graph Debug Tool is showing image alt text

Thanks all!

**Comment by surma** [comment]
🎉

**Comment by jakearchibald** [comment]
Maybe a separate issue, but do we want to use a larger image preview like we do with proxx.app?

**Pull request merged**